.vsts-pipelines/dotnet-buildtools-prereqs-ubuntu14.yml

```diff
@@ -4,14 +4,14 @@ trigger:
     - master
   paths:
     include:
-    - src/ubuntu/14*
+    - src/ubuntu/14.04/*
 pr:
```

Comment:
This comment is orthogonal to these changes.  Should these pr triggers be removed?  They are a little mis-leading.

Reply:
I could go either way.  They still can apply if we were to ever do a pull request on the internal repo.

Reply:
My preference is to clean up dead/unused code paths as they are a maintenance tax and add complexity to the code base.  In this particular case, it adds confusion as to why this is needed in addition to https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/master/.vsts-pipelines/dotnet-buildtools-prereqs-pr.yml.  If needed in the future it would be easy to add back.